examples/tanstack-start-demo/src/styles.css

```
@@ -0,0 +1,22 @@
+@tailwind base;
```

Stylelint (17.9.0)

[error] Unexpected unknown at-rule "@tailwind" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)